build/common.gni

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains common build flags for building Crosswalk. Users should
+# NOT directly import this file into their build folder.
+# See //xwalk/build/linux.gni for how to import flags for Crosswalk-Linux Build
+
+# Crosswalk specific build flags are defined in xwalk.gni. Import them.
```

Comment:
I don't see why you need to import this file, you're not using any of the values defined there. The BUILD.gn and .gni files using those values should be the ones importing it.